docs/_reference/oauth.md

```diff
@@ -172,19 +163,25 @@ slug: oauth
 <tbody>
 <tr>
 <td align="center">req</td>
-<td align="center"><code>IncomingMessage</code></td>
+<td align="center"><a href="https://nodejs.org/api/http.html#class-httpincomingmessage"><code>IncomingMessage</code></a></td>
 <td align="center">✓</td>
 <td></td>
 </tr>
 <tr>
 <td align="center">res</td>
-<td align="center"><code>ServerResponse</code></td>
+<td align="center"><a href="https://nodejs.org/api/http.html#class-httpserverresponse"><code>ServerResponse</code></a></td>
 <td align="center">✓</td>
 <td></td>
 </tr>
 <tr>
 <td align="center">options</td>
-<td align="center"><code><a href="#callbackoptions" title="">CallbackOptions</a></code></td>
+<td align="center"><code><a href="#installpathoptions" title="">InstallPathOptions</a></code></td>
```

Comment:
We would like to have this near `CallbackOptions`.

* InstallPathOptions for handleInstallPath
* CallbackOptions for handleCallback

Reply:
@seratch I don't understand? The method descriptions link to the appropriate interface used in its arguments. I'm not sure we should put the method descriptions near the interface descriptions as that breaks how we have the reference API organized: 1. InstallProvider class and its methods. 2. All other interfaces used in the class.

Reply:
@filmaj Ah, I overlooked that the items in this page are listed in alphabetical order. Never mind. In the future, we may want to generate this reference from source code comments.

Reply:
Yes absolutely !